app/Http/Controllers/DocumentController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Document;
+use App\File;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Support\Facades\DB;
+
+
+
+class DocumentController extends Controller
+{
+
+    public function store(Request $request)
+    { 
+        // Validate
+        $this->validate($request, [
+
+        // Datos Document
+        'title' => 'required|string',
+        'description' => 'required|string',
+        'author_id'=> 'required|integer',
+        'user_id'=> 'integer', 
+        'file' => 'mimes:pdf|required',
+        'directory_id' => 'integer|required'    
+
+        ]);
+
+        //Upload file
+        $fileName = $request->file('file')->getClientOriginalName();
+        $destinationPath = '/storage/app/';
```

Comment:
We should not harcode this path, maybe we could define a constant in a setting file.